ntbs-service/Models/Notification.cs

```diff
@@ -94,7 +94,11 @@ private string GetNotificationStatusString()
         }
 
         private int? CalculateSum(int? x, int? y) {
-            return x + y;
+            if (x == null && y == null) 
```

Comment:
If one of the values is null we would still like to calculate total. Right @ChristianMcc ?
Assuming this I added `x ?? 0 + y ?? 0`

but then I needed to check if both values are not null at the same time, otherwise the total is 0 when both values are missing